point_of_sale/posbox/index.rst

```diff
@@ -0,0 +1,538 @@
+available.
+
+When the posbox boots with a Wi-Fi adapter it will start its own Wi-Fi
+Access Point called "Posbox" you can connect to. The receipt that gets
+printed when the posbox starts will reflect this. In order to make the
+posbox connect to an already existing Wi-Fi network, go to the
+homepage of the posbox (indicated on the receipt) and go to the Wi-Fi
+configuration page. On there you can choose a network to connect
+to. Note that we only support open and WPA(2)-PSK networks. When
+connecting to a WPA-secured network, fill in the password field. The
+posbox will attempt to connect to the specified network and will print
+a new posbox status receipt after it has connected.
+
+If you plan on permanently setting up the posbox with Wi-Fi, you can
+use the "persistent" checkbox on the Wi-Fi configuration page when
+connecting to a network. This will make the network choice persist
+across reboots. This means that instead of starting up its own
+"Posbox" network it will always attempt to connect to the specified
+network after it boots.
+
+When the posbox fails to connect to a network it will fall back to
+starting it's own "Posbox" Access Point. If connection is lost with a
+Wi-Fi network after connecting to it, the Posbox will attempt to
+re-establish connection automatically.
+
+Multi-POS Configuration
+-----------------------
+
+The advised way to setup a multi Point of Sale shop is to have one
+Posbox per Point of Sale. In this case it is mandatory to manually
+specify the IP address of each Posbox in each Point of Sale. You must
+also configure your network to make sure the Posbox's IP addresses
+don't change. Please refer to your router documentation.
+
+Posboxless Guide (for advanced users)
+=====================================
+
+.. image:: media/posboxless_setup.png
+
+If you are running your Point of Sale on a Debian-based Linux
+distribution, you do not need the Posbox as you can run its software
+locally. However the installation process is not foolproof. You'll need
+at least to know how to install and run Odoo. You may also run into
+issues specific to your distribution or to your particular setup and
+hardware configuration.
+
+Drivers for the various types of supported hardware are provided as
+Odoo modules. In fact, the posbox runs an instance of Odoo that the
+Point of Sale communicates with. The instance of Odoo running on the
+posbox is very different from a 'real' Odoo instance however. It does
+not handle *any* business data (eg. POS orders), but only serves as a
+gateway between the Point of Sale and the hardware.
+
+The goal of this section will be to setup a local Odoo instance that
+behaves like the Odoo instance running on the Posbox.
+
+Image building process
+----------------------
+
+We generate the posbox images at
+``http://nightly.odoo.com/master/posbox/`` using the scripts in
+``https://github.com/odoo/odoo/tree/8.0/addons/point_of_sale/tools/posbox``. More
```

Comment:
ok that we need to sort it out, would be good to port everything to 9 as all the user doc assume we are using 9.0